spec/opportunities/user_views_all_opportunities_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+feature "user views upcoming opportunities on site" do
+
+  let!(:user) { FactoryGirl.create(:user, volunteer: false) }
+  let!(:organization) { FactoryGirl.create(:organization) }
+  let!(:org_admin) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.